src/data/general-cs-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const generalCSQuiz = [
+  {
+  Question: "Which of the following is a primary application of computer vision?",
+  Answer: "Object recognition",
+  Distractor1: "Word processing",
+  Distractor2: "Spreadsheet calculations",
+  Distractor3: "Database management",
+  Explanation: "Computer vision primarily deals with enabling machines to interpret and make decisions based on visual data. Object recognition is a fundamental application of computer vision where the system identifies objects within an image or video.",
+  Link: "https://www.g2.com/articles/object-recognition"
```

Comment:
we only accept links from freeCodeCamp resource like the new publication or youtube channel or wikipedia. 
please find a new link here